frontend_tests/node_tests/unread.js

```diff
@@ -191,15 +191,15 @@ var zero_counts = {
 
     var num_msgs = 10000;
     var i;
-    for (i = 0; i < num_msgs; ++i) {
+    for (i = 0; i < num_msgs;i+=1) {
```

Comment:
What do you mean? `for(i = 0; i < num_msgs;i+=1)` like this? Or like this `for(i = 0; i < num_msgs; i+=1)`